Jakefile.js

```diff
@@ -233,9 +233,12 @@ var builtLocalCompiler = path.join(builtLocalDirectory, compilerFilename);
     * @param generateDeclarations: true to compile using --declaration
     * @param outDir: true to compile using --outDir
     * @param keepComments: false to compile using --removeComments
+    * @param noResolve: specify whether to add ///-references or module import targets to the list of compiled files --noResolve

```

Comment:
i think it is about time we turned this to a single input with a property bag, so that not every call have to specify the 12 options with default values.